drafts/draft-20005.adoc

```diff
@@ -0,0 +1,207 @@
+
+** Within a top level JAR of the enterprise application.
+** Within a resource adapter module of the enterprise application.
+** Within the web module of the enterprise application.
+** Within an EJB module of the enterprise application.
+** Within a web application.
+
+It should be noted that JAAS Custom Login Modules require the use of a resource reference with container managed authentication.
+
+You can find out more about [Configuring a JAAS custom login module for Liberty](https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/twlp_sec_jaas.html)
+
+[#OLL]
+=== Open Liberty logging now has a date and time stamp 
+
+In Liberty, users have the option to format their server logs that appear in the console.log file in different formats, such as JSON and dev (default) formats, in their server logging configuration, using the consoleFormat logging attribute. The dev format will show messages in a basic format, where there are no timestamp or any other relevant information. It will only show the message with its log level and the message itself.
+
+For example:
+[source, xml]
+----
+consoleFormat=dev (default)
+[AUDIT ] CWWKE0001I: The server server1 has been launched.
+----
+
+This feature introduces a new option called `simple` for the consoleFormat logging server configuration attribute. This new option will configure Liberty to output logs in the same simple (default) format as the message.log file, with date/time stamps and other relevant information, to the `console.log` file or to the console (Standard-out).
+
+For example:
+
+[source, xml]
+----
+consoleFormat=simple
+[25/11/19 10:02:30:080 EST] 00000001 com.ibm.ws.kernel.launch.internal.FrameworkManager A CWWKE0001I: The server server1 has been launched.
+----
+
+To configure the Liberty logs to output logs in the new simple console format, you just have to set the following logging server configuration, in the bootstrap.properties, server.xml, or in the server.env (environment variables).
+
+==== bootstrap.properties
+`com.ibm.ws.logging.console.format=simple`
+
+==== server.xml
+`<logging consoleFormat="simple"/>`
+
+==== server.env
+`WLP_LOGGING_CONSOLE_FORMAT=simple`
+
+[#EPT]
+=== Persistent EJB Timers coordination and failover across members
+
+Prior to this feature, it was possible to partly coordinate automatic EJB persistent timers across multiple Open Liberty servers by configuring the EJB timer service to persist timers to the same database. This caused a single timer instance to be created on one of the servers but without the ability to fail over to another server if the original server stops or crashes. To enable fail over, this feature adds a new configurable attribute, `missedTaskThreshold`, which specifies the maximum amount of time that you want to allow for an execution of a persistent timer to complete before allowing another server to take over and run it instead.
```

Comment:
Regarding the question around “failover” vs “fail over”, I’m certain that one particular occurrence, “… the ability to fail over to …” must remain as is because it is being used as a verb. However, I’m not sure what the best practice is when the term is used as a noun.  Personally, I would use “failover” (and google seems to agree with that), but several message updates got through ID reviews (as well as some spell checkers) with “fail over”.  I’ll rely on your expertise here.  That said, we need to make the blog post at least be consistent in which way it uses when the term is used as a noun.  Currently, it has “failover” in the title, and uses “fail over” within the text, and that needs to be fixed so that all occurrences within the blog post are consistent.

Inconsistent usage as a noun includes:
“Persistent EJB Timers coordination and **failover** across members”
“To enable **fail over**, …”
“To use **fail over**, …”